tests/integration/fixtures/gcp_deployment/great_expectations/great_expectations.yml

```diff
@@ -13,7 +13,64 @@ config_version: 3.0
 # Datasources tell Great Expectations where your data lives and how to get it.
 # You can use the CLI command `great_expectations datasource new` to help you
 # add a new datasource. Read more at https://docs.greatexpectations.io/en/latest/reference/core_concepts/datasource.html
-datasources: {}
+datasources:
+  my_taxi_datasource:
+    execution_engine:
+      module_name: great_expectations.execution_engine
+      class_name: PandasExecutionEngine
+    module_name: great_expectations.datasource
+    class_name: Datasource
+    data_connectors:
+      default_inferred_data_connector_name:
+        module_name: great_expectations.datasource.data_connector
+        default_regex:
+          group_names:
+            - data_asset_name
+          pattern: (.*)
+        base_directory: <PATH TO YOUR LOCAL DATA>
```

Comment:
This doesn't actually work right, so what purpose does this DataConnector serve? Are we writing over this value in the integration test?

Reply:
If we don't use this in an integration test, should we move it to an "example" configs directory?

Reply:
Converting to draft for now